cfbs/utils.py

```diff
@@ -278,6 +286,18 @@ def cfbs_dir(append=None) -> str:
     return os.path.join(directory, append)
 
 
+def string_sha256(input):
+    return hashlib.sha256(input.encode("utf-8")).hexdigest()
+
+
+def file_sha256(file):
+    h = hashlib.sha256()
+
+    h.update(open(file, "rb").read())
+
+    return h.hexdigest()
```

Comment:
Nice utility functions. Not very critical in this case, but you could add some unit tests to `./tests/test_utils.py`

Reply:
Done in https://github.com/cfengine/cfbs/pull/208/commits/2768a80ff7341b1d1e7330180f1f5e66ac771605